x-pack/test/fleet_api_integration/apis/test_users.ts

```diff
@@ -73,6 +73,37 @@ export const testUsers: {
     username: 'integr_all',
     password: 'changeme',
   },
+  // for package_policy get one, bulk get with ids, get list
+  any_integr_read_policy_read: {
+    permissions: {
+      feature: {
+        fleet: ['read'],
+        siem: [
+          'minimal_all',
+          'trusted_applications_read',
+          'host_isolation_exceptions_read',
+          'blocklist_read',
+          'event_filters_read',
+          'policy_management_read',
+        ],
+      },
+      spaces: ['*'],
+    },
+    username: 'any_integr_read_policy_read',
+    password: 'changeme',
+  },
+  // for package_policy update API
+  any_integr_write_policy_write: {
```

Comment:
the name could be more specific to endpoint e.g. `endpoint_integr_write_policy`

Reply:
done 34519c6171a69fe44579f786bf89ada1fe7a7059